modules/web/css/ballerina-file-editor.css

```diff
@@ -892,7 +892,7 @@ input#package-name-input {
 }
 
 .annotation-table td{
-    height: 20px;
+    height: 18px;
```

Comment:
Shall we change it there as well? And I tested after changing it, but didn't overcome any issue.